oak/server/rust/oak_runtime/src/runtime/mod.rs

```diff
@@ -289,6 +307,31 @@ impl Runtime {
             }
         }
     }
+
+    /// Assigns a randomized reference to a [`Node`].
+    fn new_node_handle(&self) -> NodeRef {
+        let mut handle;
+
+        loop {
```

Comment:
I feel it would be sufficient / simpler to just use an incrementing (atomic) counter, similar to how Process IDs work in OSs.

Reply:
Done. Argh, I had randomized it as I think thats the direct you want to go with Channels.

Reply:
I think channels will also not be randomized, but channel **handles** (read or write) will be, since they are the ones used as capabilities.